src/action.js

```diff
@@ -0,0 +1,182 @@
+import * as os from "os";
+
+import * as github from "@actions/github";
+import * as toolCache from "@actions/tool-cache";
+import * as core from "@actions/core";
+import * as exec from "@actions/exec";
+
+// Define consts
+
+export const gruntworkOrg = "gruntwork-io";
+export const patcherRepo = "patcher-cli";
+export const patcherVersion = "v0.4.3";
```

Comment:
Should this be a param that gets passed in here? Perhaps an optional param with a default so that if the user doesn't specify it pulls latest?

Reply:
I am still not sure how this will be handled. Given we are calling the binary in the action, if the user always pulls the latest of the action, but has a fixed version of the binary, this will lead to some bugs when we release a new backwards-incompatible release.